packages/components/pie-button/README.md

```diff
@@ -19,3 +19,8 @@ Local dev server using Vite (with hot module reloading)
 ```
 yarn dev
 ```
+
+Local watcher using tsup and Vite (watches for changes and recompiles `dist`)
```

Comment:
Arguably we don't need the local `dev` anymore since running `watch` alongside the `dev` script inside `pie-storybook` does the same thing, whilst also recompiling. But could be good to keep anyway as a backup 

`watch` on its own doesn't open up a browser, it just recompiles the dist so can technically be used alongside any `dev` script if needed